### PR TITLE
Feature/review leva docs

### PR DESCRIPTION
**Pull request by martsec**

### CODE REVIEW

File: src/org/ods/util/Project.groovy [hunk 1]
```diff
@@ -483,7 +483,10 @@ class Project {
             "techSpecs": [
                 "DEMO-49"
             ],
-            "tests": []
+            "tests": [
+                "PLTFMDEV-401",
```
Comment (georgfedermann) [inline src/org/ods/util/Project.groovy:487]: tests PLTFMDEV-401 and PLTFMDEV-1046 will need references to DEMO-50, too, for referential integrity

Comment (martsec) [reply src/org/ods/util/Project.groovy:487]: True. Thanks!

File: src/org/ods/util/Project.groovy [hunk 2]
```diff
@@ -706,6 +709,9 @@ class Project {
                 "DEMO-15",
                 "DEMO-26"
             ],
+            "risks": [
+                "DEMO-40"
```
Comment (georgfedermann) [inline src/org/ods/util/Project.groovy]: DEMO-50, right?

File: src/org/ods/util/Project.groovy [hunk 3]
```diff
@@ -925,11 +931,14 @@ class Project {
...
```
Comment (georgfedermann) [inline src/org/ods/util/Project.groovy]: DEMO-50, right?

Comment (martsec) [reply src/org/ods/util/Project.groovy]: True. My bad

**Pull request merged**